### PR TITLE
Coding practice 19

### PR DESCRIPTION
**Pull request by waghmaresuyog**

### CODE REVIEW

File: simple/src/codingPractice/ReverseWords.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package codingPractice;
+
+public class ReverseWords {
+    public static String reverseWords(String sentence) {
```
Comment (Ams12) [inline simple/src/codingPractice/ReverseWords.java:4]: Give space between class and method.

**Review by Ams12 — CHANGES_REQUESTED** [review]

File: simple/src/codingPractice/ReverseWords.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    }
+
+    public static void main(String[] args) {
+        String sentence = "the sky is blue";
+        System.out.println("The original Sentence is :" + sentence);
+        String reversedWords = reverseWords(sentence);
+        System.out.println("After Replacing in reverse order ");
+        System.out.println(reversedWords);
+    }
+
```
[the remark this answers is not in the document]
Comment (waghmaresuyog) [reply simple/src/codingPractice/ReverseWords.java]: done

File: simple/src/codingPractice/ReverseWords.java (hunk 1 above)
Comment (waghmaresuyog) [reply simple/src/codingPractice/ReverseWords.java:5]: done

**Review by prashantkawa — APPROVED** [review]

**Review by anushreeSonak — APPROVED** [review]
Approving PR

**Pull request merged**